.github/workflows/CI.yml

```
@@ -35,26 +35,24 @@ jobs:
     - name: Setup integration-test-docker-environment
       uses: actions/setup-python@v2
 
+    - name: Install via apt
+      run: sudo apt-get install unixodbc unixodbc-dev libboost-date-time-dev libboost-locale-dev libboost-system-dev
+
     - name: Setup Python ${{ matrix.python }}
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python }}
```

```suggestion
    - name: Setup Python ${{ matrix.python }}
      uses: actions/setup-python@v3
      with:
        python-version: ${{ matrix.python }}
        cache: 'poetry'
```

see https://github.com/actions/setup-python#caching-packages-dependencies